### PR TITLE
Allow to ignore (un-)frozen submaps in the occupancy grid node.

### PR DESCRIPTION
**Pull request by MichaelGrupp**

### CODE REVIEW

File: cartographer_ros_msgs/msg/SubmapEntry.msg [hunk 1]
```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+bool is_frozen
```
Comment (gaschler) [inline cartographer_ros_msgs/msg/SubmapEntry.msg]: nit: move to the end of the message?

File: cartographer_ros/cartographer_ros/occupancy_grid_node_main.cc [hunk 2]
```diff
@@ -40,6 +40,10 @@
 DEFINE_double(resolution, 0.05,
               "Resolution of a grid cell in the published occupancy grid.");
 DEFINE_double(publish_period_sec, 1.0, "OccupancyGrid publishing period.");
+DEFINE_bool(ignore_frozen_submaps, false,
```
Comment (gaschler) [inline cartographer_ros/cartographer_ros/occupancy_grid_node_main.cc]: Perhaps it is less negations and easier to read with "show_(un)frozen_submaps" and default true?

**Comment by MichaelGrupp** [comment]
Negated. While we're at it, I also added the possibility to change the topic if desired. Hope it's ok to sneak this small change in ;)

For example, one could publish frozen and unfrozen maps simultaneously on different topics with two occ. grid nodes for debugging.

**Comment by gaschler** [comment]
@wally-the-cartographer merge

**Comment by wally-the-cartographer** [comment]
Merge requested by authorized user gaschler. Merge queue now has a length of 2.

**Pull request merged**